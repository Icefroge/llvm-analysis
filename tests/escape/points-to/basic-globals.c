int a;
int * p;
int ** pp;

void f() {
  pp = &p;
  p = &a;
}
