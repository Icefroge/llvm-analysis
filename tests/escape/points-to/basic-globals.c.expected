fromList [ ("@p", fromList ["@a"])
         , ("@pp", fromList ["@p"])
         ]